openscap_daemon/oscap_helpers.py

```diff
@@ -443,3 +443,24 @@ def get_status_from_exit_code(exit_code):
     "generate_report_for_result",
     "get_status_from_exit_code"
 ]
+
+
+def generate_fix_from_result(config, results_path):
+    if not os.path.exists(results_path):
+        raise RuntimeError("Can't generate fix for scan result. Expected "
+                           "results XML at '%s' but the file doesn't exist."
+                           % results_path)
+    try:
+        tree = ElementTree.parse(results_path)
+        root = tree.getroot()
+        ns = {"xccdf": "http://checklists.nist.gov/xccdf/1.2"}
+        test_result = root.find(".//xccdf:TestResult", ns)
+        result_id = test_result.attrib["id"]
+    except:
+        raise RuntimeError("Could not find result ID in " + results_path + "\n")
+    args = [config.oscap_path, "xccdf", "generate", "fix",
+            "--result-id", result_id,
+            "--template", "urn:xccdf:fix:script:sh",
```

Comment:
make this configurable, add support to other parts of openscap-daemon.

`oscapd-cli result 1 3 bash_fix` etc...